projects/beluga/index.js

```diff
@@ -2,9 +2,9 @@ const utils = require('../helper/utils');
 
 async function fetch() {
   let data = await utils.fetchURL('https://api.beluga.fi/tvl')
-  return data.data.tvl;
+  return {polygon: data.data.Polygon, fantom: data.data.Fantom};
 }
 
 module.exports = {
   fetch
```

Comment:
this fetch no longe works, can you update it to:
    const utils = require('../helper/utils');

    function fetch(chain) {
      return async () => utils.fetchURL('https://api.beluga.fi/tvl').then(d => ({ 'usd-coin':  d.data[chain] }))
    }

    module.exports = {
      timetravel: false,
      polygon: {
        tvl: fetch('Polygon')
      },
      fantom: {
        tvl: fetch('Fantom')
      },
    }

Reply:
Done